src/consensus/vote_listener.zig

```diff
@@ -881,24 +892,13 @@ fn trackNewVotesAndNotifyConfirmations(
             });
 
             if (is_gossip_vote and is_new and stake > 0) {
-                senders.gossip_verified_vote_hash.send(.{ vote_pubkey, slot, hash }) catch |err| {
-                    logger.err().logf(
-                        "{s}: gossip verified vote hash couldn't send: " ++
-                            ".{{ .vote_pubkey = {f}, .slot = {}, .hash = {f} }}",
-                        .{ @errorName(err), vote_pubkey, slot, hash },
-                    );
-                };
+                const gossip_verified_vote_hash = senders.gossip_verified_vote_hashes;
+                try gossip_verified_vote_hash.append(allocator, .{ vote_pubkey, slot, hash });
             }
 
-            if (reached_threshold_results.isSet(0)) {
-                if (senders.duplicate_confirmed_slots) |sender| {
-                    sender.send(.{ .slot = slot, .hash = hash }) catch |err| {
-                        logger.err().logf(
-                            "{s}: duplicate confirmed slot couldn't send: {} (hash: {f})",
-                            .{ @errorName(err), slot, hash },
-                        );
-                    };
-                }
+            if (reached_threshold_results.isSet(0)) blk: {
+                const dupe_confirmed_slots = senders.duplicate_confirmed_slots orelse break :blk;
```

Comment:
Same question as https://github.com/Syndica/sig/pull/1029/files#r2500348666

Reply:
In this case it's because of the need to unwrap the optional; not sure of a better way to express this.